templates/common/azure/files/chrony.yaml

```diff
@@ -0,0 +1,50 @@
+filesystem: "root"
+mode: 0644
+path: "/etc/chrony.conf"
+contents:
+  inline: |
+    # This file is shipped by the Machine Config Operator to synchronize time RHCOS host time on azure.
```

Comment:
Looks like an extra `time` in there? Also maybe `s/azure/Azure/`.

Reply:
ah yes, will fix it.